### PR TITLE
Concent check version compatibility

### PR DESCRIPTION
**Pull request by jiivan**

### CODE REVIEW

File: golem/utils.py [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
 import logging
 import socket
 import sys
 
+import semantic_version
+
 from ethereum.utils import sha3
 
-import binascii
+logger = logging.getLogger("golem.network.concent.client")
```
Comment (maaktweluit) [inline golem/utils.py]: The name of the logger and the file do not match, why not `__name__`?

File: tests/golem/network/concent/test_concent_client.py [hunk 2]
```diff
@@ -33,6 +35,8 @@ def setUp(self):
 
     def test_message(self, post_mock):
         response = requests.Response()
+        response.headers['Concent-Golem-Messages-Version'] = \
+            golem_messages.__version__
```
Comment (maaktweluit) [inline tests/golem/network/concent/test_concent_client.py:39]: this can use the new constants you added too

**Review by maaktweluit — APPROVED** [review]
LGTM!

2 small comments

File: tests/golem/network/concent/test_concent_client.py (hunk 2 above)
Comment (jiivan) [reply tests/golem/network/concent/test_concent_client.py:39]: No, no. This is HTTP response header and requires a string.

File: golem/utils.py (hunk 1 above)
Comment (jiivan) [reply golem/utils.py]: Good catch of Mr Paste Copy code :)

**Pull request merged**